tox.ini

```diff
@@ -21,7 +21,9 @@ ignore=
     # Comments
     E266,
     # Type Annotations
-    ANN002,ANN003,ANN101,ANN102,ANN204,ANN206
+    ANN002,ANN003,ANN101,ANN102,ANN204,ANN206,
+    # Some more "S" errors
+    S605,S607
```

Comment:
Instead of ignoring pep8 errors, pls try to fix them.

Reply:
Tried my best, these errors are happening on the same line and noqa cannot interpret them, leading to fail before commit.
You could replicate this by removing this line and running pre-commit in CamReader.py. The line system("clear") will show the error.

Reply:
@dhananjaylatkar can you refer em example for  ANN001 Missing type annotation for function argument 'w'

Reply:
i ran it on your old code. I think you added `int` there. It is fixed now